core/utils/constants.py

```diff
@@ -33,6 +36,7 @@ class Extension(Enum):
     Extension.PGVECTOR_HNSW: {'m': 32, 'ef_construction': 128, 'ef': 10},
     Extension.LANTERN: {'m': 32, 'ef_construction': 128, 'ef': 10},
     Extension.NEON: {'m': 32, 'ef_construction': 128, 'ef': 10},
+    Cloud.PINECONE: { 'name': '', 'metric': 'cosine', 'pods': 1, 'replicas': 1, 'pod_type': 'p2' },
```

Comment:
I think all the other indices are using L2 by default, and sift uses the L2 metric for ground truth.

Reply:
There was an option of euclidean distance which seems to be l2-norm, but I think in our index it is l2 squared. Will it work as expected?